### ISSUE TITLE
null reference exception when options class property has not setter.

### ISSUE THREAD

**Issue by Ivony** [commandlineparser/commandline#380]
For Excample:

```CSharp
  class Program
  {

    private class Options
    {
      [Value( 0, MetaName = "Test", Default = 0 )]
      public int TestValue{ get; }
    }

    static void Main( string[] args )
    {
      Parser.Default.ParseArguments<Options>( args );//throw NullReferenceException.
    }
  }
```

**Comment by SIkebe** [comment]
@ericnewton76 Is this issue already fixed in 2.5.0?

**Comment by moh-hassan** [comment]
The issue is fixed in [v2.5.0](https://github.com/commandlineparser/commandline/pull/394/commits/d9efdac38848e007e93da76305b322d03875193e#diff-5ccde8f7110f8ef6f87c2a1397757c83R1137)
With unit test [OptionClass_IsImmutable_HasNoCtor()](https://github.com/commandlineparser/commandline/blob/master/tests/CommandLine.Tests/Unit/Core/InstanceBuilderTests.cs#L1154)


When the options class property has not a setter and has no Ctor, it throw `InvalidOperationException`
